app/models/public_body.rb

```diff
@@ -67,16 +67,16 @@ class ImportCSVDryRun < StandardError ; end
 
   validates :request_email,
             not_nil: { message: N_("Request email can't be nil") }
-  validates :last_edit_comment,
-            not_nil: { message: N_("Last edit comment can't be nil") }
-  validates :home_page, not_nil: { message: N_("Home page can't be nil") }
-  validates :notes, not_nil: { message: N_("Notes can't be nil") }
-  validates :publication_scheme,
-            not_nil: { message: N_("Publication scheme can't be nil") }
-  validates :disclosure_log,
-            not_nil: { message: N_("Disclosure log can't be nil") }
-
-  validates_uniqueness_of :short_name, :message => N_("Short name is already taken"), :allow_blank => true
+  #validates :last_edit_comment,
+            #not_nil: { message: N_("Last edit comment can't be nil") }
+  #validates :home_page, not_nil: { message: N_("Home page can't be nil") }
+  #validates :notes, not_nil: { message: N_("Notes can't be nil") }
+  #validates :publication_scheme,
+            #not_nil: { message: N_("Publication scheme can't be nil") }
+  #validates :disclosure_log,
+            #not_nil: { message: N_("Disclosure log can't be nil") }
+
+  validates_uniqueness_of :short_name, :message => N_("Short name is already taken")#, :allow_blank => true
```

Comment:
Need to remove commented code; might as well reduce line length at the same time if its still too long